index.js

```diff
@@ -188,7 +164,7 @@ function sendNotification(endpoint, params) {
         options.headers['Authorization'] = 'Bearer ' + jwt;
         var key = 'p256ecdsa=' + urlBase64.encode(vapid.publicKey);
         if (options.headers['Crypto-Key']) {
-          options.headers['Crypto-Key'] += ',' + key;
+          options.headers['Crypto-Key'] += ';' + key;
```

Comment:
From the implementation of `Autopush` it looks like, even if it assumed different values to be separated by comma, it doesn't break if you use a semicolon instead.
@jrconlin tested and confirmed this, so I think we can use the semicolon for the time being.